### PR TITLE
Switched from UglifyJsPlugin to TerserPlugin

### PR DESCRIPTION
**Pull request by thegnuu**
Hi @alonrbar ,

As discussed in #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
   "dependencies": {
     "jszip": "3.2.2",
     "lodash.get": "4.4.2",
+    "terser-webpack-plugin": "^2.2.1",
```
Comment (alonrbar) [inline package.json]: Should be in devDependencies, also please use exact version (no ^)

**Comment by alonrbar** [comment]
Other than the comment I made it looks good, but I want to test it on my machine before merging. Will get to this hopefully today and if not in the next couple of days.

**Comment by thegnuu** [comment]
You are totally right, sorry i missed that one.

**Comment by alonrbar** [comment]
Sharing with you that I have moved to Rollup instead of Webpack and gave up on uglifying completely.
The uglified version was actually ignored since package.json "main" prop was pointing to the un-uglified version...

**Pull request merged**